lib/workout.js

```diff
@@ -23,8 +23,9 @@ var parseLine = function parseLine(line) {
             if (date > moment()) {
                 date.subtract(1, 'week');
             }
-        } else if (!word.match(/[^0-9\/]/) && word.match(/[,\/]/)) {
```

Comment:
There are no tests for dates with hyphens. Hyphen-dates don't pass these regexes so I don't think any hyphen-date worked before this patch...?